Sources/ApodiniDeployBuildSupport/DeploymentProvider.swift

```diff
@@ -24,6 +24,21 @@ public struct DeploymentProviderID: RawRepresentable, Hashable, Equatable, Codab
     }
 }
 
+public struct DeploymentStorage: MemoryStorage {
```

Comment:
Nice to add this static storage! Maybe add the 'Memory' to the `DeploymentStorage`, so `DeploymentMemoryStorage`. Therefore the purpose is immediately clear. But of course that's just my opinion, if you think otherwise we can definitely leave this as it is.